nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -1351,66 +1389,81 @@ in the file specified in `nifi.login.identity.provider.configuration.file`. Sett
 |nifi.security.ocsp.responder.certificate|This is the location of the OCSP responder certificate if one is being used. It is blank by default.
 |====
 
-*Cluster Common Properties* +
+==== Cluster Common Properties
 
-When setting up a NiFi cluster, these properties should be configured the same way on both the cluster manager and the nodes.
+When setting up a NiFi cluster, these properties should be configured the same way on all nodes.
 
 |====
 |*Property*|*Description*
-|nifi.cluster.protocol.heartbeat.interval|The interval at which nodes should emit heartbeats to the cluster manager. The default value is 5 sec.
+|nifi.cluster.protocol.heartbeat.interval|The interval at which nodes should emit heartbeats to the Cluster Coordinator. The default value is 5 sec.
 |nifi.cluster.protocol.is.secure|This indicates whether cluster communications are secure. The default value is _false_.
-|nifi.cluster.protocol.socket.timeout|The amount of time to wait for a cluster protocol socket to be established before trying again. The default value is 30 sec.
-|nifi.cluster.protocol.connection.handshake.timeout|The amount of time to wait for a node to connect to the cluster. The default value is 45 sec.
+|nifi.cluster.node.event.history.size|When the state of a node in the cluster is changed, an event is generated
+and can be viewed in the Cluster page. This value indicates how many events to keep in memory for each node.
+|nifi.cluster.node.connection.timeout|When connecting to another node in the cluster, specifies how long this node should wait before considering
+the connection a failure.
+|nifi.cluster.node.read.timeout|When communicating with another node in the cluster, specifies how long this node should wait to receive information
+from the remote node before considering the communication with the node a failure.
+|nifi.cluster.firewall.file|The location of the node firewall file. This is a file that may be used to list all the nodes that are allowed to connect
```

Comment:
Where is the format of the firewall file described? How does it perform hostname verification?